CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing guidelines
+
+Thank you for your interest in the project. We look forward to your
```

Comment:
It'd be good to refer to docker's contributing guidelines here, in the same way we do for docker-compose. https://github.com/docker/compose/blob/master/CONTRIBUTING.md#contributing-to-compose